src/DataTransferObjects/Responses/FulfillmentOutbound/CreateFulfillmentOrderResponse.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Jasara\AmznSPA\DataTransferObjects\Responses\FulfillmentOutbound;
+
+use Jasara\AmznSPA\DataTransferObjects\Responses\BaseResponse;
+use Jasara\AmznSPA\DataTransferObjects\Schemas\ErrorListSchema;
+use Jasara\AmznSPA\DataTransferObjects\Schemas\ErrorSchema;
+use Spatie\DataTransferObject\Attributes\CastWith;
+use Spatie\DataTransferObject\Casters\ArrayCaster;
+
+class CreateFulfillmentOrderResponse extends BaseResponse
+{
+    #[CastWith(ArrayCaster::class, itemType: ErrorSchema::class)]
+    public ?ErrorListSchema $payload;
```

Comment:
There is already an error list in the base response, you don't need to re-add it here